firefox-ios/firefox-ios-tests/Tests/XCUITests/SettingsTests.swift

```diff
@@ -144,24 +144,31 @@ class SettingsTests: BaseTestCase {
         checkShowImages(showImages: true)
     }
 
-    // https://mozilla.testrail.io/index.php?/cases/view/2306808
+    // https://mozilla.testrail.io/index.php?/cases/view/2951435
     // Smoketest
     func testSettingsOptionSubtitles() {
         mozWaitForElementToExist(app.buttons[AccessibilityIdentifiers.Toolbar.settingsMenuButton])
         navigator.nowAt(NewTabScreen)
         navigator.goto(SettingsScreen)
         let table = app.tables.element(boundBy: 0)
         let settingsQuery = AccessibilityIdentifiers.Settings.self
+        let settingsTitle = app.staticTexts["Settings"]
+        let doneBtn = app.buttons["Done"]
```

Comment:
Nit: use the full name since it is not a very long variable name and would be easier to read: doneButton